crates/swc_css_parser/src/parser/at_rule.rs

```diff
@@ -1113,7 +1118,30 @@ where
                     }
                 };
 
-                // TODO validate comparison
+                let is_valid_operator = match left_comparison {
```

Comment:
I think `clippy` will be angry at this.

Reply:
hm, let's check it, what is command run clippy?

Reply:
You can run `cargo clippy`

Reply:
clippy is fine :smile: